src/Aspire.Hosting.Apache.Pulsar/PulsarBuilderExtensions.cs

```diff
@@ -21,22 +21,22 @@ public static class PulsarBuilderExtensions
     /// </remarks>
     /// <param name="builder">The <see cref="IDistributedApplicationBuilder"/>.</param>
     /// <param name="name">The name of the resource. This name will be used as the endpoint name when referenced in dependency.</param>
-    /// <param name="servicePort">The service port that the underlying container is bound to when running locally.</param>
-    /// <param name="brokerPort">The broker port that the underlying container is bound to when running locally.</param>
+    /// <param name="targetPort">The host port of the Pulsar service that the underlying container is bound to when running locally.</param>
```

Comment:
Why is this configurable?

Reply:
Yes, no specific reason.

Made it more in sync with Kafka, Rabbit:
Allowed only port config for pulsar service
Allowed only port config for pulsar manager FE
Updated tests